infra/charts/feast-feature-server/values.yaml

```diff
@@ -34,6 +37,10 @@ securityContext: {}
 service:
   type: ClusterIP
   port: 80
+  online : 6566
+  ui : 8888
+  offline : 8815
+  registry : 6570
 
```

Comment:
why are these being added here? this field/value should not be modified as, again, it serves a different purpose entirely. these ports, instead, should be set directly in the service and deployment conditions.

Reply:
removed from here and added within deployment